RoaringBitmap/src/main/java/org/roaringbitmap/longlong/Roaring64Bitmap.java

```diff
@@ -740,4 +666,229 @@ public Roaring64Bitmap clone() {
       throw new RuntimeException("fail to clone thorough the ser/deser", e);
     }
   }
+  
+  private class ForwardPeekableIterator implements PeekableLongIterator {
```

Comment:
The `ForwardPeekableIterator` and `ReversePeekableIterator` could share a common abstract class.

Reply:
have added the common abstract class